### DISCUSSION TITLE
Option for HFR clips

### DISCUSSION THREAD

**Discussion by movingwater** [latenitefilms/BRAWToolbox#53] [Feature Requests]
It would be great have an option for HFR clips to be imported as high frame rate. If you record in 25fps and toggle the HFR mode the camera records in 50 or 60fps but stores the clip as 25fps. 
So it would be nice to import these clips in HFR mode. Otherwise audio and video doesn't match.

**Comment by latenitefilms** [comment]
So you're recording with a 25fps base frame rate, in 50fps HFR, and you want to speed up the 50fps footage, so it appears to play at 25fps in the timeline, so that it is in sync with the audio?

How do you imagine this working? I assume you'd still want the control in Final Cut Pro to still adjust the speed, so you just want us to add a speed effect to the clips?

One idea could be to import both a Synchronised Clip/Multicam Clip as it's currently doing, then another version that is "sped up" so that the audio is in sync? We did this on [E-High](https://fcp.co/final-cut-pro/articles/2226-chris-hocking-from-latenite-films-explains-how-his-team-used-final-cut-pro-x-to-edit-mix-grade-and-deliver-the-much-anticipated-series-neighbours-erinsborough-high-for-fremantle-australia) - so we had a slow-mo version of the clip and a realtime version of the clip.

Thoughts?

**Reply by movingwater** [reply]
Exactly. It would be nice if the imported clip would be shown as a 50fps clip. 
It the moment a clip looks like that:
![Screenshot 2023-03-14 at 14 28 16](https://user-images.githubusercontent.com/8091819/225015994-5e1bf115-218c-4bd1-9fb2-ec0f9c717889.png)

One version would be enough for me if its possible to retime it in the timeline.

In Resolve I normally create a smart bin where I can show all the HFR clips (Camara FPS is 50) and then go to "clip attributes.." and change the framerate to 50fps. 

I hope this makes sense?
